File: app/src/main/java/com/my/televip/virtuals/ui/LaunchActivity.java
package com.my.televip.virtuals.ui;

import android.widget.FrameLayout;

import com.my.televip.obfuscate.ArgsResolver;
import com.my.televip.obfuscate.Obfuscate;

import de.robv.android.xposed.XposedHelpers;

public class LaunchActivity {

    Object launchActivity;
    public FrameLayout frameLayout;

    public LaunchActivity(Object obj){
       launchActivity = obj;
       frameLayout = (FrameLayout) XposedHelpers.getObjectField(obj, Obfuscate.getFieldName("LaunchActivity","frameLayout"));
    }

}

File: app/src/main/java/com/my/televip/virtuals/ui/LaunchActivity.kt
package com.my.televip.virtuals.ui

import android.widget.FrameLayout
import com.my.televip.obfuscate.Obfuscate
import de.robv.android.xposed.XposedHelpers

class LaunchActivity(obj: Any?) {

    @JvmField
    var frameLayout: FrameLayout = XposedHelpers.getObjectField(obj, Obfuscate.getFieldName("LaunchActivity", "actionFrameLayout")) as FrameLayout
}
